Release `v1.3.0`

robbot006 opened the pull request:
This is a draft of changelog for the `v1.3.0` release.
Here you can find all related translations changes: [v1.2.1...master](https://github.com/flarum-lang/finnish/compare/v1.2.1...master).

> ### ⚠️ Do not merge this pull request manually ⚠️
> 
> You should **[approve](https://github.com/rob006-software/flarum-translations/wiki/How-to-approve-release-pull-request)** this pull request instead. After approving, I will automatically update it (release date needs to be adjusted), merge, and tag a new release. You can read more about the process on [forum](https://discuss.flarum.org/d/20807-simplify-translation-process-with-weblate/76).

<details>
<summary>Show release notes preview</summary>

## v1.3.0

<!-- release-notes-begin -->
**General changes**:

* Updated Day.js translations.
* Updated Flarum core translations.


**Added support for new extensions**:

* [`afrux/forum-widgets-core`](https://github.com/afrux/forum-widgets-core)
* [`afrux/online-users-widget`](https://github.com/afrux/online-users-widget)
* [`askvortsov/flarum-categories`](https://github.com/askvortsov1/flarum-categories)
* [`clarkwinkelmann/flarum-ext-post-bookmarks`](https://github.com/clarkwinkelmann/flarum-ext-post-bookmarks)


**Updated translations for extensions**:

* [`flarum/statistics`](https://github.com/flarum/statistics)
* [`fof/analytics`](https://github.com/FriendsOfFlarum/analytics)
* [`fof/merge-discussions`](https://github.com/FriendsOfFlarum/merge-discussions)


All changes: [v1.2.1...v1.3.0](https://github.com/flarum-lang/finnish/compare/v1.2.1...v1.3.0).



<!-- release-notes-end -->

</details>

rob006 commented:
@MarkoKaartinen Just as a note, this is an automated PR generated by bot. The idea is to simplify release process for language pack maintainers - all you need is to approve this pull request and bot will handle the rest. You can read more about the process at forum: https://discuss.flarum.org/d/20807-simplify-translation-process-with-weblate/76 . If you don't want to use this feature, let me know.

MarkoKaartinen commented:
> @MarkoKaartinen Just as a note, this is an automated PR generated by bot. The idea is to simplify release process for language pack maintainers - all you need is to approve this pull request and bot will handle the rest. You can read more about the process at forum: https://discuss.flarum.org/d/20807-simplify-translation-process-with-weblate/76 . If you don't want to use this feature, let me know.

Thanks for the information. I will read about this next week after my vacation.

robbot006 commented:
Success! Now you can announce new release on [forum](https://discuss.flarum.org/d/28810-finnish-language-pack):

~~~markdown
## Version [`v1.3.0`](https://github.com/flarum-lang/finnish/releases/tag/v1.3.0)

**General changes**:

* Updated Day.js translations.
* Updated Flarum core translations.


**Added support for new extensions**:

* [`afrux/forum-widgets-core`](https://github.com/afrux/forum-widgets-core)
* [`afrux/online-users-widget`](https://github.com/afrux/online-users-widget)
* [`askvortsov/flarum-categories`](https://github.com/askvortsov1/flarum-categories)
* [`clarkwinkelmann/flarum-ext-post-bookmarks`](https://github.com/clarkwinkelmann/flarum-ext-post-bookmarks)


**Updated translations for extensions**:

* [`flarum/statistics`](https://github.com/flarum/statistics)
* [`fof/analytics`](https://github.com/FriendsOfFlarum/analytics)
* [`fof/merge-discussions`](https://github.com/FriendsOfFlarum/merge-discussions)


All changes: [v1.2.1...v1.3.0](https://github.com/flarum-lang/finnish/compare/v1.2.1...v1.3.0).

In order to update run these commands:

```console
composer update flarum-lang/finnish
php flarum cache:clear
```

~~~
